.github/workflows/osrm-backend.yml

```diff
@@ -264,15 +264,6 @@ jobs:
             CXXCOMPILER: g++-8
             CXXFLAGS: -Wno-cast-function-type
 
-          - name: gcc-7-release
```

Comment:
We probably can add gcc-12 instead in separate PR

Reply:
yes, good call with the separate PR. It seems the code doesn't compile cleanly with GCC12, yet. Limited testing started to throw a fair amount of compile errors.